src/types/inc/utils.hpp

```diff
@@ -53,6 +53,7 @@ namespace Microsoft::Console::Utils
     bool HexToUint(const wchar_t wch, unsigned int& value) noexcept;
     bool StringToUint(const std::wstring_view wstr, unsigned int& value);
     std::vector<std::wstring_view> SplitString(const std::wstring_view wstr, const wchar_t delimiter) noexcept;
+    std::vector<std::wstring> SplitResourceStringWithPlaceholders(std::wstring resourceString, std::unordered_map<size_t, std::wstring> placeholderToStringMap);

```

Comment:
can `resourceString` be a `wstring_view`? We're making a lot of unnecessary copies, and string views have very cheap `substr()`